README.md

```diff
@@ -55,18 +55,20 @@ In short, to get a major feature added to Amundsen, one must first get the RFC m
 1. RFCs rarely go through this process unchanged, especially as alternatives and drawbacks are shown. You can make edits, big and small, to the RFC to clarify or change the design, but make changes as new commits to the pull request, and leave a comment on the pull request explaining your changes. Specifically, do not squash or rebase commits after they are visible on the pull request.
 1. RFCs that are candidates for inclusion in Amundsen will enter a **"Final Comment Period (FCP)"**, along with a disposition for the RFC (merge, close, or postpone).
-   - This step is taken **when enough of the tradeoffs have been discussed** so that the maintainers are in a position to make a decision. That does not require consensus amongst all participants in the RFC thread. However, the argument supporting the disposition on the RFC needs to have already been clearly articulated. There should not be a strong consensus against that position outside of the maintainer's group.
-   - The FCP period lasts **seven days**, and the beginning of this period will be signaled with a comment and tag on the RFC's pull request.
-   - For RFCs with lengthy discussion, the motion to FCP is usually preceded by a summary comment trying to lay out the current state of the discussion and major tradeoffs/points of disagreement.
-   - Maintainers will ping the `#amundsen` channel about the RFC to attract the community's attention.
-   - An RFC can be modified based upon feedback from the team and community. Significant modifications may trigger a new final comment period.
-1. An RFC may be rejected by the team after the public discussion has been settled and comments have been made to summarize the rationale for rejection. A member of the maintainer's group should then close the RFCs associated pull request. Also, an RFC author may withdraw their RFC by closing it themselves.
-1. An RFC may be accepted at the close of its final comment period. A team member will merge the RFCs associated pull request, at which point the RFC will become 'active'.
-1. Maintainers will meet every quarter and choose three or five active RFCs based on popularity and alignment with project vision and goals. Those selected items become part of the Mid-term goals on our roadmap.
-
+    - This step is taken **when enough of the tradeoffs have been discussed** so that the maintainers are in a position to make a decision. That does not require consensus amongst all participants in the RFC thread. However, the argument supporting the disposition on the RFC needs to have already been clearly articulated. There should not be a strong consensus against that position outside of the maintainer's group.
+    - The FCP period lasts **seven days**, and the beginning of this period will be signaled with a comment and tag on the RFC's pull request.
```

Comment:
could we make it shorter (e.g 48 hours, or 72 hours)? Often times, we forget to add the label and it takes so long to merge the pr.

Reply:
In reality this is where the conversation is supposed to happen, before it is the RFC creation process.
IMO 48 or 72 hrs wouldn´t allow us to have many back and forths with people located in different timezones

Reply:
important to note that this changeset isn't modifying this length, this just got picked up as part of a whitespace change -- suggest reading this diff with `?w=1`